prbt_support/launch/robot.launch

```diff
@@ -1,9 +1,25 @@
 <?xml version="1.0"?>
 <launch>	
-
+  <!-- gripper e.g. pg70. See documentation of meta-package prbt_grippers -->
   <arg name="gripper" default="" />
+
+  <!-- load description to the parameter server. If you wrote your own xacro model, you can load it yourself
+       and set load_robot_description to false -->
   <arg name="load_robot_description" default="true" />
+
+  <!-- safe-torque-off-input from the safety controller is read to do a stop1 -->
+  <!-- The signal is read via modbus and you can configure the modbus server ip -->
   <arg name="sto" default="pnoz"/>
+  <arg name="sto_modbus_server_ip" default="169.254.60.1" />
+
+  <!-- set the name of the config file for canopen_motor_node. If you want to change settings, copy the yaml
+       file into your package and set canopen_config to your new config file path -->
+  <arg name="canopen_config" default="$(find prbt_support)/config/manipulator_driver.yaml" />
+
+  <!-- manipulator_controller.yaml defines different types of controllers and you can select which ones
+       are automatically started -->
+  <arg name="spawn_controllers" default="manipulator_joint_trajectory_controller manipulator_joint_state_controller gripper_joint_trajectory_controller" />
```

Comment:
I would like to have a working default configuration both with and without gripper.
This leads to an error when launching without gripper:

> [ERROR] [WallTime: 1549522528.320749033 /opt/ros/kinetic/include/joint_trajectory_controller/joint_trajectory_controller_impl.h 115]: Could not find joint 'prbt_gripper_finger_left_joint' in URDF model.
[ERROR] [WallTime: 1549522528.320817794 /opt/ros/kinetic/include/controller_interface/controller.h 123]: Failed to initialize the controller
[ERROR] [WallTime: 1549522528.320833873 /tmp/binarydeb/ros-kinetic-controller-manager-0.13.3/src/controller_manager.cpp 247]: Initializing controller 'gripper_joint_trajectory_controller' failed

Reply:
@martiniil Thanks for the hint. Please re-test.